lib/usd/translators/shading/usdFileTextureWriter.cpp

```diff
@@ -77,7 +78,7 @@ TF_DEFINE_PRIVATE_TOKENS(
     _tokens,
 
     // UsdPrimvarReader_float2 Prim Name
-    ((PrimvarReaderShaderName, "TexCoordReader"))
+    ((PrimvarReaderShaderName, "shared_TexCoordReader"))
```

Comment:
Could changing this name affects loading of existing files?

Reply:
No. The name of the place2dTexture node is not restored when importing (and it was not previously written, as you can see).